client/driver/rkt.go

```diff
@@ -731,6 +926,77 @@ func (d *RktDriver) Open(ctx *ExecContext, handleID string) (DriverHandle, error
 	return h, nil
 }
 
+func (d *RktDriver) fetchImage(img lastditch.RuntimeImage, globalArgs []string) (string, error) {
+	var version string
```

Comment:
You can just do version := "latest" and then override if set like you do below. WDYT?